dotcom-rendering/src/components/Card/components/PlayIcon.tsx

```diff
@@ -12,12 +12,28 @@ const sizes = {
 	xlarge: { button: 80, icon: 72 },
 } as const satisfies Record<string, { button: number; icon: number }>;
 
-const iconWrapperStyles = css`
+const iconWrapperStyles = (
+	size: PlayButtonSize,
+	sizeOnMobile: Extract<PlayButtonSize, 'small' | 'large'>,
+) => css`
 	display: flex; /* Fixes the div mis-sizing itself */
 	position: absolute;
 	top: 50%;
 	left: 50%;
-	transform: translate(-50%, -50%);
+	margin-top: -${sizes[sizeOnMobile].button / 2}px;
+	margin-left: -${sizes[sizeOnMobile].button / 2}px;
+	${from.tablet} {
+		margin-top: -${sizes[size].button / 2}px;
+		margin-left: -${sizes[size].button / 2}px;
+	}
```

Comment:
I think we should keep the `transform` as it's relative to the element and therefore will work for any size.

In comparison, `top` and `left` are relative to the parent.

Reply:
Thanks max, you're correct. We translate the icon on hover too to make sure the element kept the right position on scale.